src/bp/nlu-core/entities/microsoft-extractor/microsoft-extractor.test.ts

```diff
@@ -0,0 +1,263 @@
+import 'bluebird-global'
+import _ from 'lodash'
+import path from 'path'
+import { unlinkSync } from 'fs'
+import { MicrosoftEntityExtractor } from '.'
+import { SystemEntityCacheManager } from '../entity-cache-manager'
+
+describe('Microsoft Extract Multiple', () => {
+  let microsoft: MicrosoftEntityExtractor
+  let testCachePath = path.join(process.APP_DATA_PATH || '', 'cache', 'testCache.json')
+  beforeAll(() => {
+    const duckCache = new SystemEntityCacheManager(testCachePath, true)
```

Comment:
try mocking this one and make sure only "cache miss" utterances are actually sent to microsoft entity extractor.

also make sure the cache is called.

If I'm an innocent developper who happends to walk by your code and fully remove the cache, all test will still pass. This should not be the caseé